src/test/java/com/fryrank/dal/ReviewDALTests.java

```diff
@@ -2,12 +2,15 @@
 
 import com.fryrank.model.GetAllReviewsOutput;
 import com.fryrank.model.Review;
+import org.bson.Document;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.mockito.InjectMocks;
 import org.mockito.Mock;
+import org.mockito.Mockito;
 import org.mockito.junit.MockitoJUnitRunner;
 import org.springframework.data.mongodb.core.FindAndReplaceOptions;
+import org.springframework.data.mongodb.core.aggregation.*;
```

Comment:
star import :0

Reply:
Done in [0df6bdc](https://github.com/NickPriv/FryRankBackend/pull/83/commits/0df6bdcd56fe31a5a0cffd72820e1d88b492fe1c).